README.md

```diff
@@ -204,8 +204,8 @@ post the results. `reducers` take data, modify it, and update the internal
 `state`.
 
 Communication of data is done using objects called `actions`. Each `action` has
-any number of properties for data, and a unique `type` that can trigger
-properties on the models.
+any number of properties for `data`, and a unique `actionName` that can trigger
```

Comment:
Perhaps change this to be somewhere along the lines of:
> Each action consists of a unique `actionName` and an optional payload of `data`, which can be any value.

Perhaps would make it more clear?